lib/mnemonic.js

```diff
@@ -141,8 +141,9 @@ Mnemonic.isValid = function(mnemonic, wordlist) {
  * @returns {boolean}
  */
 Mnemonic._belongsToWordlist = function(mnemonic, wordlist) {
-  var word = unorm.nfkd(mnemonic).split(' ')[0];
-  return wordlist.indexOf(word) !== -1; // only checks for a word
+  var splitM = unorm.nfkd(mnemonic).split(' ');
+  var found = splitM.filter(function(m) { return wordlist.indexOf(m) != -1 });
```

Comment:
I think a `for` loop would be better here as once a word isn't in the `wordlist`, we don't need to continue and can return false.